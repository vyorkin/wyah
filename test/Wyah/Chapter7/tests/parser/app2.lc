let g = f (f (f (f (f x))));
