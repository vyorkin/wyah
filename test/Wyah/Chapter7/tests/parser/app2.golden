Program [Decl "g" (EApp (EVar (Var {unVar = "f"})) (EApp (EVar (Var {unVar = "f"})) (EApp (EVar (Var {unVar = "f"})) (EApp (EVar (Var {unVar = "f"})) (EApp (EVar (Var {unVar = "f"})) (EVar (Var {unVar = "x"})))))))]